packages/pigeon/test/dart_generator_test.dart

```diff
@@ -401,6 +445,51 @@ void main() {
     expect(code, contains('EnumClass doSomething(EnumClass arg0);'));
   });
 
+  test('flutter non-nullable enum argument with enum class', () {
```

Comment:
Why? This is testing the type itself, not a value. A type can't be non-null (except in the sense of not being the special null type in languages where there is one, but that's not what this is testing), it can only be non-nullable.

Reply:
I didn't realize that dart uses `non-nullable`.  That's confusing since other languages (java, objc) use "nullable/nonnull".